src/cf-forms/src/cf-forms.less

```diff
@@ -11,40 +11,40 @@
 // https://github.com/18F/web-design-standards/blob/staging/src/stylesheets/core/_variables.scss)
 
 // .a-text-input borders
-@input-border:                  #919395; // @gray-60
-@input-border__hover:           #0072ce; // @pacific;
-@input-border__focused:         #0072ce; // @pacific;
-@input-border__active:          #0072ce; // @pacific;
-@input-border__error:           #d14124; // @red;
-@input-border__warning:         #ff9e1b; // @gold;
-@input-border__success:         #20aa3f; // @green;
-@input-border__selected:        #046b99; // $color-primary-alt-darkest
+@input-border:                  @gray-60;
+@input-border__hover:           @pacific;
+@input-border__focused:         @pacific;
+@input-border__active:          @pacific;
+@input-border__error:           @red;
+@input-border__warning:         @gold;
+@input-border__success:         @green;
+@input-border__selected:        @pacific;
 
 // .a-text-input backgrounds
-@input-bg:                      #ffffff;
-@input-bg__active:              #0072ce; // @pacific;
-@input-bg__disabled:            #f7f8f9; // @gray-5;
+@input-bg:                      @white;
+@input-bg__active:              @pacific;
```

Comment:
This is the background color that radio buttons have when in a selected state. I updated the variable name to use 'selected' instead of 'active' to clarify that.